README.md

```diff
@@ -47,6 +47,12 @@ Please see the [changelog
 document](https://github.com/mwclient/mwclient/blob/master/CHANGELOG.md)
 for a list of changes.
 
+mwclient was originally written by Bryan Tong Minh. It was maintained
+for many years by Dan Michael O. Heggø. It is currently maintained by
+Waldir Pimenta, Marc Trölitzsch, Adam Williamson and Megan Cutrofello.
```

Comment:
Thanks for adding this additional context here! However, as I [mentioned in #280](https://github.com/mwclient/mwclient/issues/280#issuecomment-1913388227), I am not actually currently maintaining mwclient, and likely won't be anytime soon, so I would be wary of adding my name here as such. I think it might be more accurate to list me in the same sentence as danmichaelo (even though he did pretty much all the maintaining, and what I did was mostly housekeeping, and generally being his sidekick :smile:).

Reply:
there, how's that? @marcfrederick @RheingoldRiver are you folks OK with being listed as current maintainers? thanks!